src/setupTests.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I think the new version of CRA comes with react-testing-library, and this file is what they're leaning towards for testing setup (duh). Because of that, we might need to figure out if we can merge that somehow with our testing-utils file.